### ISSUE TITLE
Voxceleb evaluation

### ISSUE THREAD

**Issue by mrpep** [asappresearch/slue-toolkit#15]
Hi, I have some doubts regarding which data can be used for pretraining the models. I plan to do a mixture of self-supervised and supervised pretraining and I wanted to know:

- Can I use Voxceleb1 audios for pretraining? I would only use the speaker id and nationality labels as supervision, and some self-supervision (without labels), so my model would be agnostic about the sentiment annotations, but maybe could have some advantage to differentiate the Voxceleb1 speakers during finetuning, specially if there is an imbalance in the sentiments by speaker.
- Did you follow the same original dev/test splits from Voxceleb1? I am pretraining only with the dev split, so if the sentiment analysis task is evaluated only on the test split, it would not be a problem. Am I right?

**Comment by sshon-asapp** [comment]
You can use any voxceleb1 audio if it in original dev set. (Original Voxceleb1 consisted of dev, test set) 

FYI, 
slue-voxceleb `fine-tune` set came from original Voxceleb1 dev set
slue-voxceleb `dev` and `test` set came from original Voxceleb1 test set
